hbw/lib/hbw/logger.rb

```diff
@@ -1,9 +1,14 @@
 module HBW
   module Logger
-
-    # create widget logger if needed
     def logger
-      Rails.logger
+      Yell.new do |l|
+        if Rails.env.production?
+          l.adapter :datefile, Homs::Application.config.app.fetch(:logs).fetch(:production).fetch(:hbw_log), :level => Yell.level.lte(:warn)
```

Comment:
Why don't you move
```
Homs::Application.config.app.fetch(:logs)
```
to another method? It would shorten the line and make code DRYer.